db/data_migration/20150716152627_add_static_eu_logo_to_regional_development_funding.rb

```diff
@@ -0,0 +1,25 @@
+slugs = [
+  'erdf-programmes-and-resources',
+  'erdf-programmes-progress-and-achievements',
+  'applying-for-erdf-funding',
+  'erdf-national-guidance',
+  'previous-erdf-programmes-retaining-documents',
+  'european-territorial-cooperation-programmes',
+  'reuniting-europe-programme-turkey'
+]
+logo_url = "https://assets.digital.cabinet-office.gov.uk/media/55a7cc72ed915d5374000001/erdf-logo.png"
+
+puts "Adding logos to ERDF editions"
+slugs.each do |slug|
+  document = Document.find_by(slug: slug)
```

Comment:
You should be passing a `document_type` here as well.  slugs are only unique per document_type.